src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_prior.py

```diff
@@ -62,16 +62,16 @@ class WuerstchenPriorPipelineOutput(BaseOutput):
     image_embeddings: Union[torch.Tensor, np.ndarray]
 
 
-class WuerstchenPriorPipeline(DiffusionPipeline, LoraLoaderMixin):
+class WuerstchenPriorPipeline(DiffusionPipeline, StableDiffusionLoraLoaderMixin):
```

Comment:
If it inherits from the `StableDiffusionLoraLoaderMixin` the `self._lora_loadable_modules` attributed will be `["unet", "text_encoder"]` which doesn't apply here.

Reply:
Handled in [f4bc25f](https://github.com/huggingface/diffusers/pull/8774/commits/f4bc25f6055fa8e0a69bbda384c67e98ab731cef)